scripts/apollo_clean.sh

```diff
@@ -76,9 +87,10 @@ function clean() {
 
   local answer
   typeset -l answer
-  warning "All the files under ${LOG_DIR} and ${CORE_DIR} will be removed."
-  warning "Do you want to continue (Y/n)?"
+  warning "Do you want to remove all log files under \"${LOG_DIR}\"" \
+    "and core dump files under \"${CORE_DIR}\" (Y/n)?"
   answer=$(read_one_char_from_stdin)
+  info "Your choice is: ${answer}"
   if [ "${answer}" == "y" ]; then
```

Comment:
Independent flags to control log/coredump files removal via cmdline.

E.g. `apollo_clean.sh --bazel=expunge --docs --cores`